dspace-xmlui/src/main/java/cz/cuni/mff/ufal/dspace/app/xmlui/aspect/administrative/ControlPanelReplicationTabHelper.java

```diff
@@ -166,6 +166,36 @@ public static void executeCommand(Division div, Request request, Context context
 			action = "list_replicas";
 		}
 		
+		if(action.equals("Retrieve")) {
+			Map<String, String> params = request.getParameters();
+			
+			ArrayList<String> toret = new ArrayList<String>();
+	
+			for (String key : params.keySet()) {
+				if (key.startsWith(delete_prefix)) {
```

Comment:
Why use delete_prefix when retrieving? Probably the checkbox function changed from delete to mark for download. This name will be confusing to anyone reading the code next time...